misc/futures-bounded/src/lib.rs

```diff
@@ -25,4 +30,15 @@ impl fmt::Display for Timeout {
     }
 }
 
+/// Error of a future pushing
+#[derive(PartialEq, Debug)]
+pub enum PushError<T> {
+    /// The length of the set is equal to the capacity
+    BeyondCapacity(T),
+    /// The set already contained an item with this key.
```

Comment:
```suggestion
    /// The map already contained an item with this key.
```

Isn't this relevant for the _map_ only?